mailpoet/lib/Migrations/Migration_20230215_050813.php

```diff
@@ -0,0 +1,73 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Migrations;
+
+use MailPoet\Migrator\Migration;
+
+class Migration_20230215_050813 extends Migration {
+  public function run(): void {
+    $this->subjectsMigration();
+  }
+
+  private function subjectsMigration(): void {
+    $this->createTable('automation_run_subjects', [
```

Comment:
There seems to be no primary key in this table. Do we already have any such tables? I'm just thinking, it might be safer to add an autogenerated ID column...

Reply:
75dd778ec